src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = () => (
+  <div className="App">
+    <>
+      <Display />
+      <ButtonPanel />
+    </>
+  </div>
+);
```

Comment:
Here you were supposed to **ONLY** have `React.Fragment` (<>)  tag enclosing the two components, not a `<div>`. So line `6` and `11` wasn't necessary 
[[see milestone 2 requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png)]

Reply:
Coming back to `milestone4` you have to import `calculate.js` in this `<App>` component